### PR TITLE
Adding containername as a flag

### LINKED ISSUE #507: ConatinerName should be given as a flag in run_daemon.sh
**What would you like to be added**:
IssueContainerName
Here by default the name of container is always edgecore(github.com/kubeedge/kubeedge/build/edge/run_daemon.sh). The script should be able to take container name as a flag

**Why is this needed**:
This is required to run multiple containerized edgcores in a single host.

### PR DESCRIPTION
**Pull request by lidiyag**
**What type of PR is this?**
kind feature
**What this PR does / why we need it**:
This PR takes modifies the script to take containername as a flag
**Which issue(s) this PR fixes**:
Fixes #507

### CODE REVIEW

**Comment by lidiyag** [comment]
cc @rohitsardesai83 @sids-b @edisonxiang @pavan187

File: build/edge/run_daemon.sh [hunk 1]
```diff
@@ -164,7 +164,7 @@ docker_only_run_edge(){
     edgehubWebsocketUrl=wss://${cloudhub}/e632aba927ea4ac2b575ec1603d56f10/${edgename}/events 
     image=${image:-"kubeedge/edgecore:latest"}
 
-    docker run -d --name edgecore --restart always \
+    docker run -d --name ${containername} --restart always \
```
Comment (m1093782566) [inline build/edge/run_daemon.sh]: What's the default conatinername?

Comment (lidiyag) [reply build/edge/run_daemon.sh]: edgecore

Comment (sids-b) [reply build/edge/run_daemon.sh]: Please add a default name like we have added for other parameters. edgecore can be default name as it was earlier

**Pull request merged**